src/components/projects/ProjectBox.jsx

```diff
@@ -1,8 +1,125 @@
+// React Router Imports
+import { Link as ReactRouterLink } from 'react-router-dom';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Chip from '@mui/material/Chip';
+import Grid from '@mui/material/Grid';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+// Custom Imports
+import projectsList from './projectsList';
+
 const ProjectBox = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+  return (
+    <Box
+      as="section"
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center'
+      }}
+    >
+      <Typography variant="h2" sx={{ m: 5, textAlign: 'center' }}>
+        Our Projects
+      </Typography>
+      {projectsList
+        .slice(0, 2)
+        .map(({ index, title, description, status, logo, links, techStack }) => (
+          <Box
+            key={title}
+            sx={{
+              mb: { xs: 5, sm: 15 },
+              p: 5,
+              background:
+                index % 2 === 0
+                  ? 'linear-gradient(270deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
+                  : 'linear-gradient(90deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)',
+              borderRadius: '25px',
+              minWidth: { xs: '95vw', sm: '80vw' },
+              maxWidth: { sm: '85vw' }
+            }}
+          >
+            <Grid container spacing={{ xs: 1, md: 10 }}>
+              <Grid item xs={12} md={6} order={{ xs: 0, md: index % 2 === 0 ? 2 : 1 }}>
+                <CardMedia
+                  component="img"
+                  image={logo ?? null}
+                  alt={`${title} logo`}
+                  sx={{
+                    p: 3
+                  }}
+                />
+                <Stack
+                  direction="row"
+                  justifyContent="space-evenly"
+                  spacing={{ xs: 0, md: 2 }}
+                  sx={{ pt: '1rem' }}
+                >
+                  {links.map(({ href, icon }) => (
+                    <Link
+                      component={ReactRouterLink}
+                      key={href}
+                      href={href}
+                      alt={`${title} logo`}
+                      target="_blank"
+                      rel="noopener"
+                      sx={{
+                        color: '#000',
+                        display: 'flex',
+                        '&:hover': {
+                          color: 'primary.main'
+                        }
+                      }}
+                    >
+                      {/* TODO: Fix icons not rendering */}
```

Comment:
I know I've ran into this issue in a project of mine in the past, so I'll try to examine that code and see if I can figure it out again.

Reply:
Let me know please because this part in particular has been vexing

Reply:
#### So I dug into the project that I did this before (storing the icons in objects) and it looked like this:

<img width="540" alt="Screenshot 2023-11-05 at 2 22 29 PM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/2dd4c27c-f18e-49af-88f6-7450d935165c">

#### Then used in component:

<img width="848" alt="Screenshot 2023-11-05 at 2 23 22 PM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/7ee2dd4b-0a09-40c8-8040-16826e548e75">

But, I realized this is FontAwesome and not ReactIcons.  As you can see they are used differently. You just use the generic `<FontAwesomeIcon>` element and pass the icon to `icon=`.  So, I guess I'm back to the drawing board to figure it out for our codebase.

But I remember taking forever to figure out the syntax above to result in this:

https://github.com/codeforpdx/codepdx_website/assets/71395931/3830a0b5-47ba-4386-9aa4-f0f85891c2ab